ansible/playbook.yml

```
@@ -0,0 +1,200 @@
+        vps_user: "{{ env_vars.VPS_USER | default('django') }}"
+        db_name: "{{ env_vars.DB_NAME | default('education_website') }}"
+        db_user: "{{ env_vars.DB_USER | default('root') }}"
+        db_password: "{{ env_vars.DB_PASSWORD | default('change_me') }}"
+        project_name: "{{ env_vars.PROJECT_NAME | default('education-website') }}"
+        git_repo: "{{ env_vars.GIT_REPO | default(git_repo) }}"
+        git_branch: "{{ env_vars.GIT_BRANCH | default(git_branch) }}"
+        allowed_hosts: "{{ env_vars.ALLOWED_HOSTS | default('localhost,127.0.0.1') }}"
+        csrf_trusted_origins: "{{ env_vars.CSRF_TRUSTED_ORIGINS | default('localhost,127.0.0.1') }}"
+
+    - name: Ensure application user exists
+      user: { name: "{{ vps_user }}", system: yes, create_home: yes, shell: /bin/bash }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)